bears/__init__.py

```diff
@@ -6,7 +6,7 @@
 
 import os
 import sys
-
+import logging
```

Comment:
logging should be first, so that standard library imports are in alpha order.  PEP8